common/transport/amqp/src/amqp.ts

```diff
@@ -239,8 +477,9 @@ export class Amqp {
           },
           attachReceiverLink: (endpoint: string, linkOptions: any, done: GenericAmqpBaseCallback<ReceiverLink>): void => {
             debug('creating receiver link for: ' + endpoint);
-            this._receivers[endpoint] = new ReceiverLink(endpoint, linkOptions, this._amqp);
+            this._receivers[endpoint] = new ReceiverLink(endpoint, linkOptions, this._rheaSession);
             const permanentErrorHandler = (err) => {
+              debug('receiver link invokes perm error handler for attachReceiverLink: ' + endpoint + ': ' + err.toString());
```

Comment:
>perm [](start = 43, length = 4)

nit: permanent?

Reply:
Gone.

---
In reply to: [209133104](https://github.com/Azure/azure-iot-sdk-node/pull/344#discussion_r209133104) [](ancestors = 209133104)